### PR TITLE
report the key error when user is using action

### PR DESCRIPTION
**Pull request by t-yuminghe**
https://github.com/Azure/autorest.az/issues/290
finish the issue

### CODE REVIEW

File: src/plugins/azgenerator/templates/generated/CliActions.ts [hunk 1]
```diff
@@ -139,8 +141,11 @@ function GetAction(model: CodeModelAz, actionName: string, param: Parameter, key
                 }
                 ifkv = "elif";
             } while (model.SelectNextMethodParameter());
+            output.push("            " + 'else:');
+            output.push("                raise CLIError('usage error: {} [WRONG KEY] You only have the following keys to use : " + submethodparameters.join(", ") + ". And your key : '.format(option_string)" + " + kl + ' is invalid')");
         }
         model.ExitSubMethodParameters();
+
```
Comment (LianwMS) [inline src/plugins/azgenerator/templates/generated/CliActions.ts:148]: remove

**Review by LianwMS — APPROVED** [review]
:shipit:

File: src/plugins/azgenerator/templates/generated/CliActions.ts (hunk 1 above)
Comment (t-yuminghe) [reply src/plugins/azgenerator/templates/generated/CliActions.ts:148]: ok I initiate a new pr

**Comment by qiaozha** [comment]
no use

**Pull request closed without merging**